### DISCUSSION TITLE
Unable to select type of proof.

### DISCUSSION THREAD

**Discussion by amazign** [community/community#125196] [GitHub Education]
Hi
I have encountered the same issue while applying for GitHub education student pack as many others, in that the camera cannot be switched from the front camera, and I no longer have the option to upload a file. However, I managed to take a semi-legible photo of my proof of enrolment letter with the front camera, but once it was taken, the image covered the dropdown menu to select which type of proof I've uploaded. I'm beyond frustrated with this portal, one bug or glitch is annoying and can be worked around, but it's essentially unusable to me. Does anyone know if you can apply via email or through some other channel, as I'm at my wits end.
Thanks

![image](https://github.com/community/community/assets/74781590/2343719b-89fe-4129-ba3b-ddecd76db331)

**Comment by cmobley001** [comment]
Same issue, and now I am blocked for too many verification attempts. I opened up a support ticket and attached student proof...you might try that route. Google Github Student Support to find the right link.

**Comment by amazign** [comment]
The workaround I found was selecting type of proof before taking the photo,
then submitting. Hopefully the proof type stayed selected as I couldn't see
under the photo. Just curious, how many attempts did you get before you got
blocked? If this one gets rejected I'll follow you're advice re support
ticket. Thanks for the response

**Reply by polyankaglade** [reply]
For me there is no field to even choose the type of proof from, I tried all the device sizes and looked into the page code, there is nothing else to select except two photos and a reason for being far.
